epics-service/src/main/java/edu/gemini/epics/EpicsService.java

```diff
@@ -27,17 +27,21 @@
                     "([01]?\\d\\d?|2[0-4]\\d|25[0-5])$";
 
     private String _addressList;
-    
+
     // Timeout in seconds
     private double ioTimeout = 1.0;
 
+    // Number of read retries
+    private int readRetries = 0;
+
     private CAJContext _ctx;
 
-    public EpicsService(String addressList, double ioTimeout) {
+    public EpicsService(String addressList, double ioTimeout, int readRetries) {
```

Comment:
The currently running GMPs don't need this, should we default this to 0?

Reply:
And avoid having to change GPI and GHOST code. Good idea.